pest/src/error.rs

```diff
@@ -39,15 +40,18 @@ pub struct Error<R> {
 
 /// Different kinds of parsing errors.
 #[derive(Clone, Debug, Eq, Hash, PartialEq)]
-pub enum ErrorVariant<R> {
+#[cfg_attr(feature = "std", derive(thiserror::Error))]
+pub enum ErrorVariant<R: RuleType> {
     /// Generated parsing error with expected and unexpected `Rule`s
+    #[cfg_attr(feature = "std", error("{}", self.message()))]
```

Comment:
Perhaps to distinguish the variants, this could be:
```suggestion
    #[cfg_attr(feature = "std", error("parsing error: {}", self.message()))]
```